### ISSUE TITLE
`_type` is not returned with Image Metadata

### ISSUE THREAD

**Issue by b3nk3** [FormidableLabs/groqd#83]
Hi there, I've been playing around with groqd today (Amazing - 5⭐) and immediately ran into a problem.

You have the following in [`sanityImage.ts`](https://github.com/FormidableLabs/groqd/blob/main/src/sanityImage.ts#L62)
> `_type: schemas.literal("sanity.imageDimensions"),`

After a few tries I realised, that this field is not returned. (despite some docs on sanity.io)

On their reference however, it is corresponding to how my recent install behaves. [See here](https://www.sanity.io/docs/image-type#df3f768ce073)

**Comment by gksander** [comment]
Thanks for reporting! Admittedly, I just followed their docs when adding the types here. But this should be a pretty quick fix to patch this up! I'll see if I can get something in tomorrow.

**Comment by gksander** [comment]
Was just digging into this, and I'm having troubles replicating the issue – I seem to consistently get the `_type` field back on the image dimensions. Can you give me some more info on the type of image you're querying for that has that `_type` field missing?

FWIW I ran this query in a vision dashboard to check for images with missing `_type` on dimensions:

```
*[_type == "sanity.imageAsset" && defined(metadata.dimensions._type)]
```

and didn't get any results.

**Comment by b3nk3** [comment]
Hiya, sorry, busy day at work here.

Doesn't no results mean it cannot find an image asset where `_type` is defined?

I ran the same query on my Vision - on their movie sample setup and indeed, I get no results. but as soon as i removed `._type` from the query I received the results.

I believe, if you want to find assets, where there is no `_type` field defined, you need to negate the query

```
*[_type == "sanity.imageAsset" && !defined(metadata.dimensions._type)]
```

So basically, you have validated it 😀 by not getting any results.

--------

I still think we perhaps should not remove it from schema creation, but mark it optional, so if there is a weird internal Sanity thing where it may or may not be included, would still allow this to work?

**Comment by b3nk3** [comment]
Okay, I also did a bit more digging - I created my setup from a dataset provided by Sanity... aaand I think when that got loaded it lacked the `_type` field.

I've gone ahead and uploaded an image manually and that has the `_type`.

Despite this, I still think it could be optional. I've created a PR for this. What you do think? #86
